<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <!-- Default dialog size -->
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Utilities</string>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>6</pointsize>
   </font>
  </property>

  <!-- Main vertical layout -->
  <layout class="QVBoxLayout" name="verticalLayout">

    <!-- Label -->
    <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Training Folder</string>
       </property>
      </widget>
    </item>

    <!-- LineEdit + Browse Button side by side -->
    <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
          <widget class="QLineEdit" name="lineEdit_AnnotationImagesFolder">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="toolTip">
            <string>This field is pre-populated and cannot be changed at this time.</string>
           </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="pushButton_BrowseAnnotationImages">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
        </item>
      </layout>
    </item>

    <!-- Listbox -->
    <item>
      <widget class="QListWidget" name="listWidget_Folders">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Displays a list of folders for selection.</string>
       </property>
      </widget>
    </item>

    <!-- OK/Cancel buttons -->
    <item>
      <widget class="QDialogButtonBox" name="buttonBox_ok_cancel">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
    </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
